src/Builders/AdditionalParametersBuilder.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace PhpMx\Builders;
+
+use Exception;
+
+class AdditionalParametersBuilder
+{
+    private array $blocks = [];
+
+    public static function create(): self

```

Comment:
We need a private constructor ? in order to avoid two way to create a new instance, I think this pattern is called named constructor, here is a reference it's an old video but i think it's a good video https://www.youtube.com/watch?v=J0SFLG5B3wo :p

Reply:
Nice suggestion, but the private constructor is not required, this semantic cosntructor allows the user to create and chain methods immediately.

Instead of:
```
$builder = new AdditionalParametersBuilder();
$builder->addText($text);
$builder->build($text);
```

We can directly:
```
AdditionalParametersBuilder::create()->addText($text)->build();
```

But this is open since sometimes you need to create complex messages based on conditions, IE:
```
$builder = new AdditionalParametersBuilder();
$builder->addText($text);

if ($condition) {
   $builder->addText($text);
}

$builder->build();
```